src/snippets/article-vor.v1.yaml

```diff
@@ -14,3 +14,13 @@ allOf:
         figuresPdf:
             type: string
             format: uri
+        reviewedDate:
```

Comment:
@lsh-0 a couple of optional parameters are being added to vor but we are not expecting these values to be available on the article store for now. We will be introducing these values will be handled in a similar way to how we are introducing executable version date to vor. It is a task for the future (as volume increases) to determine how to make these available in an automated way.

Reply:
Is there a coordination ticket for this?

https://github.com/elifesciences/issues/blob/master/.github/ISSUE_TEMPLATE/article--feature.md